typescript/phaser-arcade-slopes.d.ts

```diff
@@ -0,0 +1,259 @@
+declare module Phaser {
+	interface Game {
+		slopes:Phaser.Plugin.ArcadeSlopes;
+	}
+
+	module Physics.Arcade {
+		interface Body {
+			slopes:Phaser.Plugin.ArcadeSlopes;
+		}
+	}
+
+	module Plugin {
+		class ArcadeSlopes extends Phaser.Plugin {
+			static SAT:string;
+			static METROID:string;
+			enable(obj:Phaser.Sprite | Phaser.Group):void;
+			enableBody(body:Phaser.Physics.Arcade.Body):void;
+			convertTilemap(map:Phaser.Tilemap, layer:number | string | Phaser.TilemapLayer, slopeMap:Object):Phaser.Tilemap;
+			convertTilemapLayer(layer:Phaser.TilemapLayer, slopeMap:Object):Phaser.TilemapLayer;
+			collide(i:number, body:Phaser.Physics.Arcade.Body, tile:Phaser.Tile, overlapOnly:boolean):boolean;
+		}
+
+		module ArcadeSlopes {
+
+
+			class Facade {
+				factory:Phaser.Plugin.ArcadeSlopes.TileSlopeFactory;
+				solvers:Object;
+				defaultSover:number;
+				enable(obj:Phaser.Sprite | Phaser.Group):void;
+				enableBody(body:Phaser.Physics.Arcade.Body):void;
+				convertTilemap(map:Phaser.Tilemap, layer:number | string | Phaser.TilemapLayer, slopeMap:Object):Phaser.Tilemap;
+				convertTilemapLayer(layer:Phaser.TilemapLayer, slopeMap:Object):Phaser.TilemapLayer;
+				collide(i:number, body:Phaser.Physics.Arcade.Body, tile:Phaser.Tile, overlapOnly:boolean):boolean;
```

Comment:
My bad, I was recalling Phaser's collision method. :) Too many methods called `collide` to remember correctly. :grin: